.flake8

```diff
@@ -1,4 +1,16 @@
 [flake8]
-ignore =
-exclude = .git, migrations
+ignore = W503  # line break before binary operator
+exclude = .git,
+          *migrations*,
+          venv,
+          services/models/__init__.py,
+          observations/tests/test_api_read.py,    # fixtures import with *
```

Comment:
Instead of ignoring complete modules, could you ignore only specific lines? See the http://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html

Reply:
F403 is the code to ignore.